BonusCalcApi/V1/Infrastructure/Migrations/20211007105426_CreateOperative.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace V1.Infrastructure.Migrations
+{
+    public partial class CreateOperative : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "operatives",
+                columns: table => new
+                {
+                    id = table.Column<string>(type: "character varying(6)", maxLength: 6, nullable: false),
+                    name = table.Column<string>(type: "character varying(100)", maxLength: 100, nullable: false),
+                    trade = table.Column<string>(type: "character varying(10)", maxLength: 10, nullable: false),
+                    section = table.Column<string>(type: "character varying(10)", maxLength: 10, nullable: false),
+                    scheme = table.Column<string>(type: "character varying(10)", maxLength: 10, nullable: false),
+                    salary_band = table.Column<int>(nullable: false),
+                    fixed_band = table.Column<bool>(nullable: false),
+                    active = table.Column<bool>(nullable: false)
```

Comment:
Is this analogous with the is-archived field in Rh database?

Reply:
Yes, I'll switch it around